docs/index.md

```diff
@@ -2,57 +2,69 @@
+* **Test Ports** with Layer 1&2 capabilities, including:
+	- LLDP, LAG, LACP
+* Emulated **OTG Devices** with Layer 2&3 features:
+	- IPv4, IPv6 interfaces
+	- ARP, IPv6 ND
+	- BGP, IS-IS routing protocols
+* **Traffic Flows** 
+    - associated with either Test Ports, or OTG Devices
+	- expressing L2-4 properties like Ethernet, IPv4/IPv6, TCP/UDP
+	- stateless or stateful capabilities for transport protocols
+	- with implementation-specific application payload
+* Run-time **Metrics** and traffic **Capture** capabilities
+
+![Example OTG Diagram](images/otg-example-diagram.svg)
+<sub>Fig. 1. Sample diagram of the OTG configuration with emulated BGP routers, traffic flows and a Device Under Test</sub>
 
 
 ## Implementations
 
-Following tools provide OTG compliant implementations.
+To apply OTG in practice, an OTG-compatible tool, typically a Traffic Generator, is needed. There are several implementations available, and the list is growing:
  
-* *[Ixia-c Community Edition](https://ixia-c.dev)*: container-based traffic generator from Keysight. Limited to 4 ports and Traffic Flows
-* *[Keysigh Elastic Network Generator](https://www.keysight.com/us/en/products/network-test/protocol-load-test/keysight-elastic-network-generator.html)*: commercial offering of OTG implementation for a family of Keysigh/Ixia Traffic Generators
-* *[IxNetwork](https://www.keysight.com/us/en/products/network-test/protocol-load-test/ixnetwork.html)*: [snappi-ixnetwork](https://github.com/open-traffic-generator/snappi-ixnetwork) supports running OTG/snappi scripts against Keysight IxNetwork
-* *[Magna](https://github.com/openconfig/magna)*: native open-source OTG implementation from [OpenConfig project](https://openconfig.net/)
-* *[TRex](https://trex-tgn.cisco.com/)*: [snappi-trex](https://github.com/open-traffic-generator/snappi-trex) supports running OTG/snappi scripts against TRex. Supports layer 2-3 Traffic Flows
+* [**Ixia-c Community Edition**](https://ixia-c.dev): container-based traffic generator from Keysight. The Community Edition supports up to 4 Test Ports and stateless layer 2-3 Traffic Flows
+* [**Keysigh Elastic Network Generator**](https://www.keysight.com/us/en/products/network-test/protocol-load-test/keysight-elastic-network-generator.html): commercial offering of OTG implementation for a family of Keysight/Ixia Traffic Generators
+* [**IxNetwork**](https://www.keysight.com/us/en/products/network-test/protocol-load-test/ixnetwork.html): [snappi-ixnetwork](https://github.com/open-traffic-generator/snappi-ixnetwork) enables running OTG/snappi scripts with Keysight IxNetwork
+* [**Magna**](https://github.com/openconfig/magna): open-source OTG implementation from [OpenConfig project](https://openconfig.net/)
+* [**TRex**](https://trex-tgn.cisco.com/): [snappi-trex](https://github.com/open-traffic-generator/snappi-trex) enables running OTG/snappi scripts with TRex. Supports layer 2-3 Traffic Flows
 
 ## Clients
```

Comment:
Do you want to repeat contents from clients.md here?

Reply:
It is the other way. At the moment the content in `clients.md` is a stub that I copied from here.